tests/wallet/nft_wallet/test_nft_wallet.py

```diff
@@ -33,13 +33,43 @@ async def tx_in_pool(mempool: MempoolManager, tx_id: bytes32) -> bool:
     return True
 
 
+def assert_rpc_response(response, condition_func=None) -> Dict:
+    assert isinstance(response, dict)
+    assert response.get("success")
```

Comment:
Does not-success not result in an exception from the client?

Reply:
true, thanks